doc/mk_sdcard_image_domu.sh

```diff
@@ -0,0 +1,353 @@
+#!/bin/bash -e
+
+MOUNT_POINT="/tmp/mntpoint"
+CUR_STEP=1
+
+usage()
+{
+	echo "`basename "$0"` <-p image-folder> <-d image-file> [-s image-size-gb] [-u dom0|domd|domu]"
+	echo "	-p image-folder	Base daily build folder where artifacts live"
+	echo "	-d image-file	Output image file or physical device"
+	echo "	-s image-size	Optional, image size in GB"
+	echo "  -u domain	Optional, unpack the domain specified"
+
+	exit 1
+}
+
+print_step()
+{
+	local caption=$1
+	echo "###############################################################################"
+	echo "Step $CUR_STEP: $caption"
+	echo "###############################################################################"
+	((CUR_STEP++))
+}
+
+###############################################################################
+# Inflate image
+###############################################################################
+inflate_image()
+{
+	local dev=$1
+	local size_gb=$2
+
+	print_step "Inflate image"
+
+	if  [ -b "$dev" ] ; then
+		echo "Using physical block device $dev"
+		return 0
+	fi
+
+	echo "Inflating image file at $dev of size ${size_gb}GB"
+
+	local inflate=1
+	if [ -e $1 ] ; then
+		echo ""
+		read -r -p "File $dev exists, remove it? [y/N]:" yesno
+		case "$yesno" in
+		[yY])
+			sudo rm -f $dev || exit 1
+		;;
+		*)
+			echo "Reusing existing image file"
+			inflate=0
+		;;
+		esac
+	fi
+	if [[ $inflate == 1 ]] ; then
+		sudo dd if=/dev/zero of=$dev bs=1M count=$(($size_gb*1024)) || exit 1
```

Comment:
I don't think you need to use `sudo` there.

Reply:
You can crease sparse file. This will speed up process and probably will use less space:

` dd if=/dev/zero of=$dev bs=1M seek=$(($size_gb*1024)) count=1 conv=sparse`